### DISCUSSION TITLE
What would be the recommend way to not get color output in DAP communication?

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/bloop#1372] [General]
Depending on the client, they may not support color in the way they are displaying test output during a debug session. For example, using [Vimspector](https://github.com/puremourning/vimspector) via coc-metals, the output is showing the escape codes. There are ways to disable color output via the cli etc, but as far as I can tell, there isn't a way to disable the colored logging, but also it seems that the `baseBspLogger` is actually created with it turned off anyways?

https://github.com/scalacenter/bloop/blob/f9fd96b8fa655474485cfc91cead432e7cfb69bf/frontend/src/main/scala/bloop/bsp/BloopBspServices.scala#L98

I don't fully see where it's then getting turned on, or what would be the recommended way to turn color off for DAP communication.

Basically, I'm trying to avoid this:
<img width="463" alt="Screenshot 2020-09-12 at 10 26 55" src="https://user-images.githubusercontent.com/13974112/92994948-39b19a80-f4ff-11ea-9bc6-649ddee544a6.png">

Any thoughts?

**Comment by tgodzik** [comment]
I think this would be great to have as an additional option to the launcher in `LauncherMain`.
